Hi, @seanbudd  
I tested the latest build and NVDA can catch this error:
```
ERROR - brailleDisplayDrivers.seikantk.BrailleDisplayDriver.terminate (10:50:45.576) - MainThread (10400):
Seika Notetaker driver not initialized
```

There seems to be another Error and I'm not quite sure how it happened:
```
RuntimeError: wrapped C/C++ object of type BoxSizer has been deleted
ERROR - unhandled exception (10:50:45.697) - MainThread (10400):
Traceback (most recent call last):
  File "wx\core.pyc", line 3407, in <lambda>
  File "gui\settingsDialogs.pyc", line 1457, in refreshGui
```

But my configuration can be reverse as expected.

Below is the full log:
[log.txt](https://github.com/nvaccess/nvda/files/8674703/log.txt)

--- a/source/braille.py
+++ b/source/braille.py
@@ -6,6 +6,7 @@
 
 import itertools
 import os
+from sys import exc_info
 import typing
 from typing import Iterable, Union, Tuple, List, Optional
 from locale import strxfrm
@@ -2428,9 +2429,9 @@ class BrailleDisplayDriver(driverHandler.Driver):
 		# Clear the display.
 		try:
 			self.display([0] * self.numCells)
-		except Exception as e:
+		except Exception:
 			# The display driver seems to be failing, but we're terminating anyway, so just ignore it.
-			log.debugWarning(f"Display driver {self} is failing while terminating. {e}")
+			log.error(f"Display driver {self} failed to display while terminating.", exc_info=True)
 
 	#: typing information for autoproperty _get_numCells
 	numCells: int

--- a/source/brailleDisplayDrivers/seikantk.py
+++ b/source/brailleDisplayDrivers/seikantk.py
@@ -97,7 +97,6 @@ def isSeikaBluetoothDeviceMatch(match: DeviceMatch) -> bool:
 
 
 class BrailleDisplayDriver(braille.BrailleDisplayDriver):
-	_dev: Optional[hwIo.IoBase] = None
 	name = SEIKA_NAME
 	# Translators: Name of a braille display.
 	description = _("Seika Notetaker")
@@ -118,6 +117,7 @@ class BrailleDisplayDriver(braille.BrailleDisplayDriver):
 		self._hidBuffer = b""
 		self._command: typing.Optional[bytes] = None
 		self._argsLen: typing.Optional[int] = None
+		self._dev: Optional[hwIo.IoBase] = None
 
 		log.debug(f"Seika Notetaker braille driver: ({port!r})")
 		dev: typing.Optional[typing.Union[hwIo.Hid, hwIo.Serial]] = None

--- a/source/brailleDisplayDrivers/seikantk.py
+++ b/source/brailleDisplayDrivers/seikantk.py
@@ -185,17 +185,17 @@ class BrailleDisplayDriver(braille.BrailleDisplayDriver):
 		return False
 
 	def terminate(self):
-		if self._dev is None:
-			log.error("Seika Notetaker driver not initialized")
-			return
 		try:
 			super().terminate()
 		finally:
+			if self._dev is None:
+				log.debugWarning("Seika Notetaker driver not initialized when attempting to terminate")
+				return
 			self._dev.close()
 
 	def display(self, cells: List[int]):
 		if self._dev is None:
-			log.error("Seika Notetaker driver not initialized")
+			log.debugWarning("Seika Notetaker driver not initialized when attempting to display")
 			return
 		# cells will already be padded up to numCells.
 		cellBytes = SEIKA_SEND_TEXT + bytes([self.numCells]) + bytes(cells)

